### DISCUSSION TITLE
Gephi/Log4J

### DISCUSSION THREAD

**Discussion by Marv968** [gephi/gephi#2390] [Q&A]
Hello,

It looks like Gephi uses the Log4J libraries, will a fix be made to patch this vulnerability?

Best regards,

Marvin

**Comment by mbastian** [comment]
We don't use Log4J libraries AFAIK. Where did you see it?

**Reply by Marv968** [reply]
Hi Mathieu,

Sorry looked into it a bit more, as our scan picks up Log4J libraries and the JNDI class.

The JNDI class has been found here within the install of Gephi:

C:\Program Files\Gephi-0.9.2\gephi\modules\ext\org.gephi.desktop-branding\com-getsentry-raven\raven.jar\com\getsentry\raven\config\Jndilookup.class

Is this affected by the vulnerability, or is it mitigated as Gephi doesn't use Log4J?

**Reply by Marv968** [reply]
Hello, is there any update to the question above?

**Comment by eduramiba** [comment]
See https://github.com/gephi/gephi/pull/2365
We will stop using old raven client and use new sentry sdk in 0.9.3

**Reply by Marv968** [reply]
Hi Eduardo! Great, thanks for letting me know. Is there a rough ETA for the release of 0.9.3?

**Reply by eduramiba** [reply]
Hi Marvin,
It will probably be early this year but there is no date yet.
